lib/plugins/input/gelf.js

```diff
@@ -30,7 +30,7 @@ InputGELF.prototype.createServer = function () {
   this.server = gelfserver();
   this.server.on('message', function (gelf) {
     //At the moment gelf server doesn't expose rinfo
```

Comment:
What is rinfo?  Is that a typo or does it really refer to something?

Reply:
Yes, the comment is not clear 👎 . With rinfo I mean remote info I've see now this [issue] (https://github.com/wavded/graygelf/issues/17). As suggest, we can retrieve it from raw message. I'll try it to improve the PR. thanks for note